comixed-tasks/src/main/java/org/comixedproject/task/model/RescanComicsWorkerTask.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * ComiXed - A digital comic book library management application.
+ * Copyright (C) 2020, The ComiXed Project
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses>
+ */
+
+package org.comixedproject.task.model;
+
+import java.util.List;
+import lombok.extern.log4j.Log4j2;
+import org.comixedproject.model.comic.Comic;
+import org.comixedproject.service.comic.ComicService;
+import org.comixedproject.service.task.TaskService;
+import org.comixedproject.task.encoders.RescanComicWorkerTaskEncoder;
+import org.springframework.beans.factory.ObjectFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableBeanFactory;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Component;
+
+/**
+ * <code>RescanComicsWorkerTask</code> queues instances of {@link RescanComicWorkerTask}.
+ *
+ * @author Darryl L. Pierce
+ */
+@Component
+@Scope(value = ConfigurableBeanFactory.SCOPE_PROTOTYPE)
+@Log4j2
+public class RescanComicsWorkerTask extends AbstractWorkerTask {
+  private static final int MAX_RECORDS = 100;
+
+  @Autowired private ComicService comicService;
+  @Autowired private TaskService taskService;
+
+  @Autowired
+  private ObjectFactory<RescanComicWorkerTaskEncoder> rescanComicWorkerTaskEncoderObjectFactory;
+
+  @Override
+  protected String createDescription() {
+    return "Rescanning comics";
+  }
+
+  @Override
+  public void startTask() throws WorkerTaskException {
```

Comment:
WorkerTaskException is never thrown in the method.

Reply:
But the method signature that's being overridden does. To keep things consistent, I don't think we should change the signature at the implementation just because it doesn't actually throw the exception.

Reply:
ok it's true